.github/workflows/ci.yml

```
@@ -18,3 +18,6 @@ jobs:
     with:
       dotnet-version: 10.0.x
     secrets: inherit
+  permissions:
```

actionlint (1.7.12)

[error] "runs-on" section is missing in job "permissions"

(syntax-check)